webapp/src/modules/translation/locales/en.json

```
@@ -897,23 +897,23 @@
       "refresh": "Refresh"
     },
     "pick_item_as_favorite_success": {
-      "title": "Item added to Favorites",
-      "body": "{name} was successfully added to Favorites",
+      "title": "Item added to my list",
```

The phrase that was suggested by the UX team was `Wishlist` which is the name of the list, would you mind changing it?
```suggestion
      "title": "Item added to Wishlist",
```